pkg/operator/ceph/cluster/controller.go

```diff
@@ -263,6 +263,12 @@ func (c *ClusterController) onAdd(obj interface{}) {
 		return
 	}
 
+	if hasCleanupPolicy(clusterObj) {
+		logger.Warningf("skipping orchestration for new cluster object %q in namespace %q, which should not set a cleanup policy",
```

Comment:
`onAdd()` will actually be called upon operator restart even for existing clusters, so we can't simply differentiate new from old clusters. We could log the same message as from the `onUpdate()` event.